packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx

```diff
@@ -50,6 +28,12 @@ export type Props = {
 }
 
 const StxAirdrop = (props: Props & LinkDispatchPropsType) => {
+  const stxCampaign = props.userCampaignsInfoResponseList.find(
+    (campaign: CampaignInfoType) => campaign.campaignName === 'BLOCKSTACK'
+  )
+
+  if (!stxCampaign) return null
```

Comment:
If we don't want to show to users that haven't participated we can add

if (stxCampagin.userCampaignState === 'NONE') return null